pkg/manifests/manifests.go

```diff
@@ -72,5 +72,31 @@ func Get() (*Manifests, error) {
 		}
 	}
 
+	// update namespace
```

Comment:
looks obvious to me

Reply:
This was a note I left for myself and forgot to remove.
Thanks